common/lib/xmodule/xmodule/modulestore/tests/test_split_mongo_call_count.py

```diff
@@ -0,0 +1,119 @@
+"""
+Tests to verify correct number of MongoDB calls during course import/export and traversal
+when using the Split modulestore.
+"""
+
+from tempfile import mkdtemp
+from shutil import rmtree
+from unittest import TestCase
+
+from xmodule.modulestore.xml_importer import import_from_xml
+from xmodule.modulestore.xml_exporter import export_to_xml
+from xmodule.modulestore.tests.factories import check_mongo_calls
+from xmodule.modulestore.tests.test_cross_modulestore_import_export import (
+    MongoContentstoreBuilder, MixedModulestoreBuilder, VersioningModulestoreBuilder,
+    TEST_DATA_DIR
+)
+
+MIXED_SPLIT_MODULESTORE_BUILDER = MixedModulestoreBuilder([('split', VersioningModulestoreBuilder())])
+
+
+class CountMongoCallsXMLRoundtrip(TestCase):
+    """
+    This class exists to test XML import and export to/from Split.
+    """
+
+    def setUp(self):
+        super(CountMongoCallsXMLRoundtrip, self).setUp()
+        self.export_dir = mkdtemp()
+        self.addCleanup(rmtree, self.export_dir, ignore_errors=True)
+
+    def test_import_export(self):
+        # Construct the contentstore for storing the first import
+        with MongoContentstoreBuilder().build() as source_content:
+            # Construct the modulestore for storing the first import (using the previously created contentstore)
+            with MIXED_SPLIT_MODULESTORE_BUILDER.build(source_content) as source_store:
+                # Construct the contentstore for storing the second import
+                with MongoContentstoreBuilder().build() as dest_content:
+                    # Construct the modulestore for storing the second import (using the second contentstore)
+                    with MIXED_SPLIT_MODULESTORE_BUILDER.build(dest_content) as dest_store:
```

Comment:
Although my changes are Split-based only, it'd be good to set down a marker for the number of Mongo calls from old-mongo as well. I'll change it.